drawee/src/main/java/com/facebook/drawee/drawable/ScalingUtils.java

```diff
@@ -193,6 +199,19 @@ public static Matrix getTransform(
         transform.postTranslate((int) (dx + 0.5f), (int) (dy + 0.5f));
         break;
 
+      case FIT_Y:
+        scale = scaleY;
+
+        float localScaleX = scaleY;
```

Comment:
Remove those 4 lines. We need to preserve the aspect ratio which means we should not stretch the image, even if it doesn't fill the parent.